### DISCUSSION TITLE
Sponsor

### DISCUSSION THREAD

**Discussion by PranamBhat** [community/community#3363] [Other Feature Feedback, Questions, & Ideas] [answered]
How can someone "**Sponsor**" me? Is there anything I have to set-up in my **GitHub** profile?

**Comment by Zi-Tao** [comment, marked as answer]
You can add a sponsor button in your repository!
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

**Comment by D7EAD** [comment]
Hi!

Other than setting up a sponsor button in a repository, you can also have a sponsor button on your personal account. You can learn more about the requirements and how to do so at [this](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account) link.

If I've managed to help you, please consider **Marking as Answer** so others can more easily find this answer if they happen to have the same question as you. Thanks!
